packages/dds/tree/src/simple-tree/objectNode.ts

```diff
@@ -147,6 +150,14 @@ export type InsertableObjectFromSchemaRecord<
 			}
 		>;
 
+/**
+ * Helper used to remove annotations from a schema record before calling InsertableObjectFromSchemaRecord.
```

Comment:
I get an error about InsertableObjectFromSchemaRecord not being exported by the tree package if I try to make it a link even though it definitely seems to be exported. @Josmithr any idea what might be going wrong?

Reply:
Oh, nevermind - you are not linking _to_ this type, you are linking to the other one.

Reply:
Hmm, that's weird. I'm not sure what would be causing that. Happy to look at it with you if you continue running into it though :)

Reply:
Thanks, issue is fixed after I resolved the merge conflicts :)